ckanext/dcatde/harvesters/dcatde_rdf.py

```diff
@@ -120,6 +122,52 @@ def after_update(self, harvest_object, dataset_dict, temp_dict):
         return None
 
     def before_create(self, harvest_object, dataset_dict, temp_dict):
+        base_context = {'model': model, 'session': model.Session,
+                'user': self._get_user_name()}
+
+        remote_orgs = json.loads(harvest_object.job.source.config).get('remote_orgs', None)
```

Comment:
Sollte vorher in der Methode `validate_config` [https://github.com/GovDataOfficial/ckanext-dcatde/blob/873db9a980da72859f99bb4d23c9a27c74f0424a/ckanext/dcatde/harvesters/dcatde_rdf.py#L407] geprüft werden, so dass zumindest kein ungültiges JSON abgespeichert werden kann. Ansonsten könnte es hier zu einer Exception kommen.